README.md

```diff
@@ -168,65 +172,12 @@ This instance will have all the prerequisite software set up for you.
 - Download and install Maven v3.6.0: https://maven.apache.org/install.html
 - Make sure to test the installation using: ```mvn -version```
 
-### Getting the code
+### <a name="detailed-code"></a> Getting the code
 
 - Clone the repo using: ```git clone https://github.com/Elegant-Queueing/q-backend.git```
 - Make sure you're on ```master``` since it has the code from the latest release
 
-### Firebase
-NOTE: The Q team already has a dev database. To get access to it, please contact the team. To create your own database, use the following information.
-
-First, set up the database using the video: https://drive.google.com/file/d/1TMxiXX76JmJRJR1CP60grtf9WiX7aRxs/view
-
-Then, make changes to the code using the video: https://drive.google.com/file/d/1eiZPQq5OnAkD-kf628ZCYe6tTYlACvDB/view
+### <a name="detailed-firebase"></a> Firebase
+NOTE: The Q team already has a dev database. To get access to it, please contact the team. Once you have access to it, make changes to the code using the video: https://drive.google.com/file/d/1eiZPQq5OnAkD-kf628ZCYe6tTYlACvDB/view
 
-Schema:
-
-```
-companies:
-	Auto-ID
-	bio: String
-	employees: Array<String [must be valid employeeID]>
-	name: String
-	roles: Array<String [must be one of SWE, PM, DS only]>  // roles this company is recruiting for
-	website: String
-
-employees:
-	AUTO-ID
-	bio: String
-	company_id: String [must be a valid companyID]
-	email: String
-	name: String
-	role: String [must be one of SWE, PM, DS only]  // role this employee will be recruiting for
-	students: Array<String [must be valid studentID]>  // the students this employee has talked to
-	
-fairs:
-	AUTO-ID
-	companies: Array<String [must be valid companyID]>  // companies attending the fair
-	desc: String
-	start_time: Timestamp
-	end_time: Timestamp
-	name: String
-	university: String [must be valid universityID]
-	
-students:
-	AUTO-ID
-	bio: String
-	email: String
-	employees: Array<String [must be valid employeeID]>  // the employees student has talked to 
-	first_name: String
-	last_name: String
-	gpa: Number
-	grad_date: Timestamp
-	international: Boolean
-	major: String
-	
-test: no document required
-
-universities:
-	AUTO-ID
-	desc: String
-	name: String
-	past_fairs: Map<'fair_id', String [must be valid fairID]>  // the key is literally the word: fair_id
-```
-	
```

Comment:
i think you can still keep this, so that a user/dev can refer to it easily if needed. Can change the title to "Schema for reference"